readme.md

```diff
@@ -112,6 +112,41 @@ The function throws an error if anything goes wrong, which can be handled by cat
 await mdToPdf({ content: '# Hello, World' }, { dest: 'path/to/output.pdf' });
 ```
 
+#### Table of content
```

Comment:
It's called "Table of contents" in general, and usually its heading is just "Contents". Can you adjust that everywhere please?

https://en.wikipedia.org/wiki/Table_of_contents